src/Core/Setting.php

```diff
@@ -20,14 +21,25 @@
 
 class Setting
 {
+
     /**
+     *
```

Comment:
I see a lot of these changes: new lines / extra spaces / and go on. You should prevent them. They make commits unclear (I can't see your important changes quick enough) and it will mess up the code because of code standard.